apeer_dev_kit/_core.py

```diff
@@ -1,59 +1,71 @@
-﻿import os
+import os
 import json
+import logging as log
 
 from ._utility import copyfile
 
 class _core:
     def __init__(self):
+        log.basicConfig(format='%(asctime)s [ADK:%(levelname)s] %(message)s', level=log.INFO, datefmt='%Y-%m-%d %H:%M:%S')
         try:
+            log.info('Initializing')
             self._outputs = {}
             self._wfe_output_params_file = ''
             self._input_json = json.loads(os.environ['WFE_INPUT_JSON'])
+            log.info('Found environment variable WFE_INPUT_JSON to be {}'.format(self._input_json))
         except KeyError:
-            raise KeyError(
-                'Environment variable WFE_INPUT_JSON not found. Please add WFE_INPUT_JSON as an environment variale to get inputs')
+            message = 'Environment variable WFE_INPUT_JSON not found'
+            log.error(message)
+            raise KeyError(message)
 
     def _get_inputs(self):
-        """ Get the inputs"""
+        ''' Get the inputs'''
         try:
-            self._wfe_output_params_file = self._input_json.pop(
-                'WFE_output_params_file')
+            self._wfe_output_params_file = '/output/' + self._input_json.pop('WFE_output_params_file')
+            log.info('Output params will be written to {}'.format(self._wfe_output_params_file))
             return self._input_json
         except KeyError:
-            raise KeyError(
-                'Key WFE_output_params_file not found. Please add WFE_output_params_file in input json')
+            message = 'Key WFE_output_params_file not found'
+            log.error(message)
+            raise KeyError(message)
 
     def _set_output(self, key, value):
+        log.info('Set output "{}" to "{}"'.format(key, value))
```

Comment:
If we adding log.info to `_set_output()` method, then shouldn't we also log in `_set_file_output()`

Reply:
I changed `_set_file_output()` to use `_set_output()` for actually setting the output value.

Reply:
ah ok I didn't realised that